src/controller/userControllers.js

```diff
@@ -183,6 +177,53 @@ class userController {
 
         }
    }
+
+   static async getUserById(req, res){
+    try {
+      const { userId } = req.params;
+      const user = await Model.User.findOne({
```

Comment:
make the User Model a global variable and use it without always putting the Model 
ex: const { User } = Model